definitions/contested/json/State/State.json

```diff
@@ -230,5 +230,13 @@
     "Name": "Consented Order Not Approved",
     "DisplayOrder": 29,
     "TitleDisplay": "**${[CASE_REFERENCE]}  ${applicantLName}  vs ${respondentLName}**"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "generalApplication",
+    "Name": "General Application",
+    "DisplayOrder": 30,
+    "TitleDisplay": "**${[CASE_REFERENCE]}  ${applicantLName}  vs ${respondentLName}**"
```

Comment:
spacing is inconsistent - 2x 2 spaces and then 1 space between `vs` and `respondentLName` - is this as expected?

Reply:
it's inconsistently consistent with every other state